openid-4-verifiable-presentations-1_0.md

```diff
@@ -708,7 +708,7 @@ Note: In the Response Mode `direct_post` or `direct_post.jwt`, the Wallet can ch
 
 This section defines how Authorization Response containing a VP Token can be signed and/or encrypted at the application level when the Response Type value is `vp_token` or `vp_token id_token`. Encrypting the Authorization Response can prevent personal data in the Authorization Response from leaking, when the Authorization Response is returned through the front channel (e.g., the browser).
 
-To sign, or sign and encrypt the Authorization Response, implementations MAY use JWT Secured Authorization Response Mode for OAuth 2.0 (JARM) [@!JARM].
+To sign, or both sign and encrypt the Authorization Response as a Nested JWT [@!RFC7519], implementations MUST use the JWT Secured Authorization Response Mode for OAuth 2.0 (JARM) [@!JARM].
```

Comment:
```suggestion
To sign, encrypt, or both sign and encrypt the Authorization Response, implementations MUST use the JWT Secured Authorization Response Mode for OAuth 2.0 (JARM) [@!JARM].
```

without this, this PR does not solve the original issue. and nested jwt here feels redundant and confusing.

Reply:
I think this sentence is actually getting more confusing. Kristina's suggestion now says you must use the extension in this section when you're just signing a response, which would not be correct.

I would leave the scope of this sentence at dealing with the signed or signed+encrypted cases only.